data/presets/amenity/school/3-US.json

```diff
@@ -0,0 +1,24 @@
+{
+    "icon": "temaki-school",
+    "locationSet": {
+        "include": [
+            "us"
+        ]
+    },
+    "terms": [
+        "senior high school",
+        "secondary school"
+    ],
+    "geometry": [
+        "area",
+        "point"
+    ],
+    "tags": {
+        "amenity": "school",
+        "isced:level": "3"
+    },
+    "reference": {
+        "key": "isced:level"
+    },
+    "name": "High School Grounds"
```

Comment:
ISCED level 3 is officially known as “upper secondary” and covers grades 10 through 12 or, alternatively, the first three years of vocational school. This preset applies to 0.9% of public schools and 0.4% of private schools. However, 9.8% of public schools and 4.0% of private schools have “High School” or “Senior High School” (but not “Junior High School”) in their name.